src/app/modules/main/module.nim

```diff
@@ -818,7 +818,7 @@ method setActiveSection*[T](self: Module[T], item: SectionItem, skipSavingInSett
   if(item.isEmpty()):
     echo "section is empty and cannot be made as active one"
     return
-  self.controller.setActiveSection(item.id, skipSavingInSettings)
+  self.controller.setActiveSection(item.id, skipSavingInSettings = skipSavingInSettings)
```

Comment:
I don't think adding `skipSavingInSettings =` is necessary. I usually recommend doing it when the passed param is a simple arg, like a bool or a string. That way we know what is passed, but in this case, the arg has the same name as the variable :smile:

Reply:
Yeah, totally makes sense. 
Here I added a named function argument because I wanted to implicitly skip passing an empty string for the second parameter (chatId).

I can rewrite it as follows:

`self.controller.setActiveSection(item.id, "", skipSavingInSettings)`

Reply:
Aaaah, nvm then, your approach makes sense :+1: